eZ/Publish/Core/Search/Legacy/Content/Common/Gateway/CriterionHandler/Field.php

```diff
@@ -148,10 +147,19 @@ public function handle(
         $fieldsInformation = $this->getFieldsInformation($criterion->target);
 
         $subSelect = $query->subSelect();
+
         $subSelect->select(
             $this->dbHandler->quoteColumn('contentobject_id')
         )->from(
             $this->dbHandler->quoteTable('ezcontentobject_attribute')
+        )->leftJoin(
+            $this->dbHandler->quoteTable('ezkeyword_attribute_link'),
+            'ezcontentobject_attribute.id',
+           'ezkeyword_attribute_link.objectattribute_id'
+        )->leftJoin(
+            $this->dbHandler->quoteTable('ezkeyword'),
+            'ezkeyword.id',
+            'ezkeyword_attribute_link.keyword_id'
```

Comment:
none of this belongs here in the generic Field criterion, adding to that ezkeyword is deprecated and will be replaced by eztags.

Reply:
Maybe do your tests on a Field that is known to work, instead of adding a different issue to the scope here.